### DISCUSSION TITLE
Known Issues: crash when using simple terminal (st)

### DISCUSSION THREAD

**Discussion by bytewife** [cargo-generate/cargo-generate#368] [Q&A] [answered]
When I try to run any
```
cargo generate --git <url>
```
like
```
cargo generate --git https://github.com/rustwasm/wasm-pack-template
```
my terminal crashes on Ubuntu 20.04. Does cargo generate store crash logs?

**Comment by sassman** [comment]
Hi @aith,

that sounds serious. Can you give me a better idea what's that terminal crash of yours then looking like?
Is your shell ending? Is your graphical console terminating? Or is there any sort of error message that you could share?

**Comment by bytewife** [comment]
The terminal ends- unsure about the shell. I'm unable to find any error messages. Here's what it looks like:

https://user-images.githubusercontent.com/62635784/124802004-5abefe00-df0c-11eb-9c0c-4f0c64d55ea7.mp4

Edit: I should also add that this is using st (simple terminal). Using the same command with the gnome terminal works as intended)

**Comment by sassman** [comment]
thanks for the additional infos. I'm a bit clueless tbh, not sure what we can do about it.
IMO this is an issue of st handling maybe colors or other things not very well. 

Maybe I'll find the time to verify this in a virtualbox or so. But frankly, you may want to get in touch with the st maintainers, maybe they're aware of any issue (or are thankful for a bug report like this :)

In theory when a terminal emulator just dies, something might be fundamentally off.
Sorry that I cannot give you a better advice, if you find a solution feel encouraged to share it here, it might help others.

**Comment by bytewife** [comment, marked as answer]
Ah I figured it out- st crashes with the emojis shown when using 'cargo generate'
